test/console/debugger_local_test.rb

```diff
@@ -34,7 +34,7 @@ def test_raised_is_accessible_from_repl
             type "catch Exception"
             type "c"
             type "_raised"
-            assert_line_text(/#<NameError: undefined local variable or method `foo' for main:Object/)
```

Comment:
Note that the exception here is printed with `#inspect` because that's how debugger prints objects from console (pp + width limit). So it's not produced by the test framework.